src/frontend/src/utils/gantt.utils.ts

```diff
@@ -105,8 +105,12 @@ export const transformWorkPackageToGanttTask = (workPackage: WorkPackage, teamNa
   };
 };
```

Comment:
The options for teams in the gantt chart filter should just be each team (and all teams and no teams), and a project should show up if it's assigned to the selected team (regardless of what other teams it is or isn't assigned to):
<img width="158" alt="image" src="https://github.com/Northeastern-Electric-Racing/FinishLine/assets/29521172/715359e2-3f06-4a1a-a9ec-305967dfaee3">

Reply:
This would be a massive change to the code